### PR TITLE
Missing steps

### PR DESCRIPTION
**Pull request by ChrisdeG**
Please revise this document from a clean computer while many small steps are missing. I added a few.

### CODE REVIEW

File: content/howto/mobile/native-build-locally.md [hunk 1]
```diff
@@ -89,7 +91,7 @@ In the sections below you can see the basic steps to get an app up and running o
 1. After a short time the app should be synchronized and the play button (**Run Locally**) should be selectable. Select a device or create a device from the drop-down menu and click the play button (**Run Locally**) to build and install your app on the device:
 
 	{{% image_container width="250" %}}![Android Build Toolbar](attachments/native-build-locally/as-start-build.png){{% /image_container %}}
-
+If no device is available use **AVD Manager** to add a device.
```
Comment (smarki) [inline content/howto/mobile/native-build-locally.md:94]: it used to be that "running" would pop to add a device. Apparently now Android Studio will fail with a random error.
Here's the screenshot to be added
image

File: content/howto/mobile/native-build-locally.md [hunk 2]
```diff
@@ -26,6 +26,8 @@ For Android Builds:
 
 * Install [Android SDK](https://developer.android.com/studio) and [platform tools](https://developer.android.com/studio/releases/platform-tools)
 
+accept the licenses with **sdkmanager --licenses** after installing in the bin directory
```
[the remark this answers is not in the document]
Comment (smarki) [reply content/howto/mobile/native-build-locally.md:29]: I would expect installing Android Studio and following the wizard would install everything correctly

**Review by smarki — CHANGES_REQUESTED** [review]

File: content/howto/mobile/native-build-locally.md [hunk 3]
```diff
@@ -26,6 +26,8 @@ For Android Builds:
 
 * Install [Android SDK](https://developer.android.com/studio) and [platform tools](https://developer.android.com/studio/releases/platform-tools)
```
[the remark this answers is not in the document]
Comment (ConnorLand) [reply content/howto/mobile/native-build-locally.md:27]: I have made this change.

File: content/howto/mobile/native-build-locally.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ConnorLand) [reply content/howto/mobile/native-build-locally.md:29]: Made the change, noted it in the comment below this one.

File: content/howto/mobile/native-build-locally.md [hunk 4]
```diff
@@ -80,7 +82,7 @@ In the sections below you can see the basic steps to get an app up and running o
 
 	{{% image_container width="350" %}}![Android Studio](attachments/native-build-locally/as-home.png){{% /image_container %}}
 
-   Mendix native mobile apps make use of **Build Variants** to build a release app or a custom developer app. The idea of **Build Variants** is a Gradle build system concept for sharing the same codebase but delivering different experiences.
+   Mendix native mobile apps make use of **Build Variants** to build a release app or a custom developer app. The idea of **Build Variants** is a Gradle build system concept for sharing the same codebase but delivering different experiences. If the Build Variants are not visible use **View/Tool Windows/Build variants** to display them.
```
[the remark this answers is not in the document]
Comment (ConnorLand) [reply content/howto/mobile/native-build-locally.md:85]: We may have collapsible content options in the future stack. For now I am keeping the information as regular text without a screenshot, as it seems fairly clear.

File: content/howto/mobile/native-build-locally.md (hunk 1 above)
Comment (ConnorLand) [reply content/howto/mobile/native-build-locally.md:94]: Thank you, I will add the screenshot.

File: content/howto/mobile/native-build-locally.md [hunk 5]
```diff
@@ -24,7 +24,7 @@ For iOS builds:
 
 For Android Builds:
 
-* Install [Android SDK](https://developer.android.com/studio) and [platform tools](https://developer.android.com/studio/releases/platform-tools)
+* Install [Android Studio](https://developer.android.com/studio) and [platform tools](https://developer.android.com/studio/releases/platform-tools), taking care to complete the wizard in Android Studio which installs licenses and other necessary parts
```
Comment (ConnorLand) [inline content/howto/mobile/native-build-locally.md:27]: Does this look good now @stmarkidis ?

Comment (smarki) [reply content/howto/mobile/native-build-locally.md:27]: licenses in this instance, need to be accepted not installed, more like TOS for the SDKs that are installed as part of the wizard.

**Review by ConnorLand — APPROVED** [review]

**Comment by ConnorLand** [comment]
Thank you for sharing your clean computer experiences @ChrisdeG !

**Pull request merged**